### PR TITLE
Update the remaining strings after the string chill review

### PR DESCRIPTION
**Pull request by radinamatic**
### Summary

Getting in all the proposed changes from the [reference doc](https://docs.google.com/document/d/17Ax5CS4YicoDerItCnykvZrijDTg42rJ7bjqVRYJTZU/edit).

### Reviewer guidance
Everything still works OK?

### References
…

----

### Contributor Checklist

PR process:

- [X] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: kolibri/core/auth/constants/commands_errors.py [hunk 1]
```diff
@@ -14,8 +15,14 @@
 
 MESSAGES = {
     UNEXPECTED_EXCEPTION: _("Unexpected exception [{}]: {}"),
-    TOO_LONG: _("Content of cell '{}' is too long"),
-    INVALID: _("Content of cell '{}' is not valid"),
+    TOO_LONG: pgettext_lazy(
```
Comment (jredrejo) [inline kolibri/core/auth/constants/commands_errors.py:18]: @radinamatic  when using `pgettext_lazy`, the first parameter is the context and the second is the text to be translated. I know it's counter intuitive, but that's just how it is ;)

**Review by jredrejo — DISMISSED** [review]

File: kolibri/core/auth/constants/commands_errors.py (hunk 1 above)
Comment (radinamatic) [reply kolibri/core/auth/constants/commands_errors.py:18]: Ah, gotcha, had no idea... Will amend asap, thanks! :+1:

**Comment by radinamatic** [comment]
@jonboiser @indirectlylit Had to fumble a bit to resolve some conflicts and Travis errors because of the errors introduced by improper conflict resolves, but now it seems all green again. 

Please review asap otherwise more stuff will get _conflicted_ :blush:

**Review by indirectlylit — APPROVED** [review]
thank you

**Pull request merged**